### ISSUE TITLE
incompatible of package between puppet-agent and mcollective

### ISSUE THREAD

**Issue by lkthomas** [voxpupuli/puppet-mcollective#314]
apt-get install puppet-agent
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cpp-5 ipsvd libcc1-0 libisl15 libksi1 libmatrixssl1.8 libmongo-client0
  libsysfs2 libusbip0 ruby-stomp ruby-systemu
Use 'apt-get autoremove' to remove them.
The following packages will be REMOVED:
  mcollective mcollective-client mcollective-common
The following NEW packages will be installed:
  puppet-agent
0 upgraded, 1 newly installed, 3 to remove and 196 not upgraded.
Need to get 14.3 MB of archives.
After this operation, 75.6 MB of additional disk space will be used.
Do you want to continue? [Y/n] n
Abort.

# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.4 LTS
Release:	14.04
Codename:	trusty

so how come when I install puppet-agent, my mcollective have to remove?!

**Comment by petems** [comment]
Mcollective is now bundled in the puppet-agent package: https://docs.puppet.com/puppet/latest/reference/about_agent.html#release-contents-puppet-agent-1x

**Comment by lkthomas** [comment]
Yeah, I understand that, what should I change on my code so that it doesn't purge my puppet-agent?

**Comment by supernovae** [comment]
I'm going to set "manage_packages=false" and see if that fixes puppet4..  had to laugh when i ran this and puppet uninstalled itself because the mco package is marked incompatible.

**Comment by madAndroid** [comment]
We're experiencing this as well with puppet4 - main issue for us, is ensuring plugins are installed, since they don't appear to be included with the AIO package.

**Comment by khaefeli** [comment]
` manage_packages=false` helps.
maybe you can set the default for Debian to false?
